realsense2_camera/src/realsense_node_factory.cpp

```diff
@@ -128,62 +179,39 @@ void RealSenseNodeFactory::onInit()
                     {
                     if (info.was_removed(_device))
                     {
-                    ROS_FATAL("The device has been disconnected! Terminating RealSense Node...");
-                    ros::shutdown();
-                    exit(1);
+                    ROS_WARN("The device has been disconnected! Trying to reconnect to a RealSense...");
```

Comment:
We can open a separate ticket, but the indentation looks odd starting from a few lines above. Am I right? This makes review hard.